docs/charts/current_usage.md

```diff
@@ -0,0 +1,184 @@
+# Usage
```

Comment:
Should have asked on the other PR but did we want to start migrating some of the docs in the other repo now?

Reply:
We could? I can see pros and cons either way.